healthcare/api-client/v1/fhir/fhir_resources.py

```diff
@@ -0,0 +1,654 @@
+# Copyright 2018 Google LLC All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+import os
+import httplib2
+
+from googleapiclient import discovery
+from googleapiclient.http import build_http
+from oauth2client.service_account import ServiceAccountCredentials
+
+# [START healthcare_get_client]
+
+
+def get_client():
+    """Returns an authorized API client by discovering the Healthcare API and
```

Comment:
Nit: Usually, we format long docstring like the following:

```python
def get_client():
    """Returns an authorized API client.

    It assumes you set GOOGLE_APPLICATION_CREDENTIALS environment
    variable.
    """
```

Short summary, then long description.

Reply:
Done.